### PR TITLE
#161779865 implement google passport authentication 

### PR DESCRIPTION
**Pull request by V-84**
#### What does this PR do?
Implement Google Passport Authentication
#### Description of Task to be completed?
Implement Google Passport Authentication
#### How should this be manually tested?
- Clone Repo
- Run 'npm install'
- Run 'npm run db:migrate'
- Run 'npm run start'
#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?
#161779865
#### Screenshots (if appropriate)
n/a
#### Questions:
n/a

### CODE REVIEW

File: server/helpers/auth/strategies/googleStrategy.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  /**
+   * @description Generate a random password for every user authenticated
+   * @param {string} password
+   * @returns {function} hashPassword
+   */
+  static hashPassword(password) {
+    const salt = bcrypt.genSaltSync(10);
+    return bcrypt.hashSync(password, salt);
+  }
+
```
Comment (Danielshow) [inline server/helpers/auth/strategies/googleStrategy.js:25]: Good one @JohnMadakin. There is an helper function in the codebase that can do this. `server/helpers/passwordHash`

**Comment by efalayi** [comment]
@JohnMadakin @Danielshow is this PR good to go? It has not been approved

**Review by idmega2000 — APPROVED** [review]

**Review by Danielshow — APPROVED** [review]
Looks good to me

File: server/helpers/auth/strategies/googleStrategy.js (hunk 1 above)
Comment (V-84) [reply server/helpers/auth/strategies/googleStrategy.js:25]: done. thanks for your observation

**Pull request merged**